internal/cmd/postgresflex/instance/backups/update-schedule/update_schedule.go

```diff
@@ -0,0 +1,112 @@
+package updateschedule
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	cliErr "github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/client"
+	postgresflexUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex"
+)
+
+const (
+	instanceIdFlag     = "instance-id"
+	backupScheduleFlag = "backup-schedule"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	InstanceId     *string
+	BackupSchedule *string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-schedule",
+		Short: "Updates backup schedule for a specific PostgreSQL Flex instance",
+		Long:  "Updates backup schedule for a specific PostgreSQL Flex instance.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Update the backup schedule of a PostgreSQL Flex instance with ID "xxx"`,
+				"$ stackit postgresflex instance backups update-schedule --instance-id xxx --backup-schedule '6 6 * * *'"),
+		),
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			instanceLabel, err := postgresflexUtils.GetInstanceName(ctx, apiClient, model.ProjectId, *model.InstanceId)
+			if err != nil {
+				instanceLabel = *model.InstanceId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to update backup schedule of instance %q?", instanceLabel)
+				err = confirm.PromptForConfirmation(cmd, prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("update backup schedule of PostgreSQL Flex instance: %w", err)
+			}
+
+			cmd.Printf("Updated backup schedule of instance %q\n", instanceLabel)
+			return nil
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+	cmd.Flags().String(backupScheduleFlag, "", "Backup schedule, in the cron scheduling system format e.g. '0 0 * * *'")
```

Comment:
how does the API error look like if the schedule is provided in a wrong format?

Reply:
could not test this yet for PostgreSQL flex, but MongoDB Flex returns "invalid format"